### PR TITLE
Updating for v9: update dependencies and run linter

### PR DESCRIPTION
**Pull request by Ranieri93**
Hello!
Here I updated all dependencies that were outdated, and tests ran as expected.
As @jsumners mentioned, we could have a problem with deps switching to ESM-only, but to be honest I did not find a way of programmatically understanding if they are or not, so if you have any suggestions please share!

### CODE REVIEW

File: package-lock.json
Comment (jsumners) [inline package-lock.json:1]: Please do not include this file.

**Review by jsumners — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (Ranieri93) [reply package-lock.json:1]: Hey yes, that's what I thought as well. But the CI was failing complaining that the package-lock.json was missing, so I commit it again and it went through. Let me delete it

Comment (Ranieri93) [reply package-lock.json:1]: `Error: Dependencies lock file is not found in /home/runner/work/json-colorizer/json-colorizer.`
I supposed that I needed to resolve this by adding the file, but I am a complete newbie to Actions in general, so I'll wait for your suggestions!

Comment (jsumners) [reply package-lock.json:1]: The issue is this line in the CI config -- https://github.com/pinojs/json-colorizer/blob/3eff43c0ede8ede4f02027268825e30fcbd47d63/.github/workflows/ci.yaml#L29

As part of this PR, remove that line in the CI config.

**Review by jsumners — APPROVED** [review]
Looks good to me.

**Pull request merged**